src/pages/Cart/CourseCartCard.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./CourseCartCard.css";
+import { useNavigate } from "react-router-dom";
+
+const CourseCartCard = ({ product }) => {
+  const navigate = useNavigate();
+  return (
+    <article className="course-cart-card">
+      <div className="img-wrapper">
+        <img src={product.img} alt={product.title} />
+      </div>
+      <div className="content">
+        <h4 onClick={() => navigate("/course")}>{product.title}</h4>
```

Comment:
you can use Link for just navigating to another page if there is no other code to run before navigating.

Reply:
yes. Thanks for reminding